docs/cloud/automation/terraform.md

```diff
@@ -1,2 +1,244 @@
+The location varies by operating system:
+
+- Linux and macOS `~/.terraform.d/plugins`
+- Windows `%APPDATA%\terraform.d\plugins`
+
+Alternatively, the binary can be placed alongside the main `terraform` binary.
+
+You can download the provider using the following commands:
+
+- Linux:
+
+<<< @/docs/cloud/automation/snippets/download_provider_linux.sh
+ 
+- macOS:  
+
+<<< @/docs/cloud/automation/snippets/download_provider_macos.sh
+
+- Windows (PowerShell): 
+
+<<< @/docs/cloud/automation/snippets/download_provider_windows.ps1.powershell
+
+## From Source 
+
+If you prefer to install from source, use the `make install` target in this [repository][terraform github].
+
+You will need a Go 1.13+ development environment.
+
+## Provider Configuration
+
+The Event Store Cloud provider must be configured with an access token.   
+There are several additional options that may be useful.
+
+Provider configuration options are:
+
+| Option              | Environment Variable   | Description                                                                                                          |
+| :------------------ | :--------------------- | :------------------------------------------------------------------------------------------------------------------- |
+| `token`             | `ESC_TOKEN`            | *Required*, your access token for Event Store Cloud                                                                  |
+| `organization_id`   | `ESC_ORG_ID`           | *Required*, the identifier of your  Event Store Cloud organization.                                                  |
+| `url`               | `ESC_URL`              | *Optional*, the URL of the Event Store Cloud API. This defaults to the public cloud instance of Event Store Cloud    |
+| `token_store`       | `ESC_TOKEN_STORE`      | *Optional*, the location on the local filesystem of the token cache. This is shared with the Event Store Cloud CLI.  |
+                                                                                                                                                                       
+### Obtaining the `token`
+Currently this token must be created and displayed with the [esc cli][esc cli github releases] tool.  
+
+The token id displayed in the cloud console is not a valid token.
+
+The command to obtain a token from the esc cli tool is  
+`esc access tokens create --email <email>`
+
+## Resources
+
+All resources in Event Store Cloud can be provisioned using the Terraform provider.  
+Existing projects can be queried using a data source in the provider.  
+More complete samples can be found [here][terraform github samples].
+
+The following resources are available : 
+- Projects : `eventstorecloud_project`
+- Networks : `eventstorecloud_network`
+- Peering  : `eventstorecloud_peering`
+- Cluster  :  `eventstorecloud_managed_cluster`
+
+
+## eventstorecloud_project
```

Comment:
**CONCERN:** The h2 headings are used in the menu, maybe we could name them like e.g.:
```suggestion
## Creating network
```
Then we'll have more self-explanatory sections?

Reply:
internal link you mean in this page ?
the `eventstorecloud_project` is well what people expect to see in the doc while scanning it 
=> terraforms does use that convention : <provider name>_<ressource>

Reply:
If that's what they expect then it's fine, however for me as a newbie that looked like some placeholder.
I was referring to e.g. 
- https://developers.eventstore.com/cloud/use/#networking for networking, 
- https://developers.eventstore.com/cloud/intro/quick-start.html#projects for projects